users/urls.py

```diff
@@ -0,0 +1,13 @@
+"""
+Maps urls to views
+"""
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path("add/", views.AddUser.as_view(), name="Add User"),
+    path("login/", views.LoginUser.as_view(), name="Login"),
+    path("update/", views.UpdateUser.as_view(), name="Update"),
+    path("delete/", views.DeleteUser.as_view(), name="Update"),
```

Comment:
These will be changed based on - If we use `APIViewSet`

Reply:
should i change them to `APIViewSet` or let them be for the time being?

Reply:
Well if you change APIView in `views.py` to ViewSet you'll need to remove these with a single mainframe URL for the viewset with router. But this whole depends upon if we want to use APIView or ViewSet.